<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPeer</class>
 <widget class="QDialog" name="DlgPeer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QTableWidget" name="tbl_envs"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lbl_envs">
     <property name="text">
      <string>Environments</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3" colspan="2">
    <widget class="QGroupBox" name="gr_peer_info">
     <property name="title">
      <string>Peer Info</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QPushButton" name="btn_launch_console">
        <property name="text">
         <string>Launch Subutai Console</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QPushButton" name="btn_peer_on_hub">
        <property name="text">
         <string>Peer on Hub</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="lbl_peer_name">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="lbl_peer_status">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
